### PR TITLE
update java get-started doc

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
Page is https://docs.microsoft.com/en-us/azure/developer/java/sdk/get-started

Fix 2 issues.

1. `maven-archetype-quickstart` template generate `App.java`, not `AzureApp.java` https://maven.apache.org/archetypes/maven-archetype-quickstart/
2. error of "Source option 5 is no longer supported. Use 7 or later." if not configure Java version

### CODE REVIEW

File: articles/java/sdk/get-started.md [hunk 1]
```diff
@@ -72,47 +72,57 @@ This step creates a basic Maven project under the *testAzureApp* directory. Add
...
+    </plugin>
+    <plugin>
+      <groupId>org.apache.maven.plugins</groupId>
+      <artifactId>maven-compiler-plugin</artifactId>
+      <version>3.0.0</version>
+      <configuration>
+        <source>11</source>
+        <target>11</target>
+      </configuration>
+    </plugin>
```
Comment (weidongxu-microsoft) [inline articles/java/sdk/get-started.md:118]: For error of `Source option 5 is no longer supported. Use 7 or later.`

Should we use `1.8` instead? As this is the minimal Java version SDK supports.

Alternatively we can use this
```
<properties>
  <maven.compiler.source>11</maven.compiler.source>
  <maven.compiler.target>11</maven.compiler.target>
</properties>
```

File: articles/java/sdk/get-started.md [hunk 2]
```diff
@@ -129,7 +139,7 @@ import com.azure.resourcemanager.compute.models.KnownLinuxVirtualMachineImage;
 import com.azure.resourcemanager.compute.models.VirtualMachine;
 import com.azure.resourcemanager.compute.models.VirtualMachineSizeTypes;
 
-public class AzureApp {
+public class App {
```
Comment (weidongxu-microsoft) [inline articles/java/sdk/get-started.md:142]: These are for `maven-archetype-quickstart`. I am not sure whether this is behavior change of maven-archetype-quickstart
https://maven.apache.org/archetypes/maven-archetype-quickstart/

**Comment by weidongxu-microsoft** [comment]
@xboxeer @srnagar @JonathanGiles for review.

We might want to upgrade the versions of the lib, use the BOM etc.
But for this PR, I wanted to focus on things that appears having problem with latest maven and Java.

File: articles/java/sdk/get-started.md [hunk 3]
```diff
@@ -72,47 +72,57 @@ This step creates a basic Maven project under the *testAzureApp* directory. Add
...
+  <version>12.8.0</version>
 </dependency>
 <dependency>
-    <groupId>com.microsoft.sqlserver</groupId>
-    <artifactId>mssql-jdbc</artifactId>
-    <version>6.2.1.jre8</version>
+  <groupId>com.microsoft.sqlserver</groupId>
+  <artifactId>mssql-jdbc</artifactId>
+  <version>6.2.1.jre8</version>
 </dependency>
```
Comment (weidongxu-microsoft) [inline articles/java/sdk/get-started.md:93]: we might want to add `slf4j-simple`?

**Comment by KarlErickson** [comment]
Hi @weidongxu-microsoft - thanks for the pull request! However, internal contributors should submit their pull requests to the internal repo at azure-dev-docs-pr. Could you please reopen there? That way we can get full build validation and other support. Thanks!

**Pull request closed without merging**